src/apps/service/apps/de/jobs/common.clj

```diff
@@ -1,17 +1,15 @@
 (ns apps.service.apps.de.jobs.common
-  (:use [kameleon.core]
-        [kameleon.entities]
-        [kameleon.queries :only [get-user-id]]
+  (:use [apps.persistence.users :only [get-user-id]]
+        [apps.util.assertions :only [assert-not-nil]]
+        [apps.util.conversions :only [remove-nil-vals]]
```

Comment:
Heh, another case where I'd like to see `clojure-commons.core/remove-nil-values` used instead.